### DISCUSSION TITLE
Gentle fork that is easier to self-host

### DISCUSSION THREAD

**Discussion by amirouche** [commonsense/conceptnet5#329] [General]
I want to help.

I am working on a fork of the web API, that I can self-host easily.

(According to `htop`, my CPU load is less than 1 and the max is 12 (mind the absence of `%`).

Do you mind if it is backward incompatible with the conceptnet5 REST API?

\cc @rspeer @tae898 @basels @TaichiY496 @Dei6 @pyrator @unxusr

I am in the [Gitter room](https://gitter.im/commonsense/conceptnet5).

**Comment by tae898** [comment]
Hey @amirouche, 

Thanks for taking a lead. Honestly I can't say much about the compatibility, since I am not one of the core developers here. Just out of curiosity, why is your fork gonna be backward incompatible?

**Reply by amirouche** [reply]
> why is your fork gonna be backward incompatible?

The API is slightly more general, hence less fine-tuned to conceptnet use-case. 

I would like to improve the knowledge base built out of conceptnet dataset that is available at this time at https://github.com/commonsense/conceptnet5/wiki/Downloads

**Comment by God-damnit-all** [comment]
> I want to help.
> 
> I am working on a fork of the web API, that I can self-host easily.
> 
> (According to `htop`, my CPU load is less than 1 and the max is 12 (mind the absence of `%`).
> 
> Do you mind if it is backward incompatible with the conceptnet5 REST API?
> 
> \cc @rspeer @tae898 @basels @TaichiY496 @Dei6 @pyrator @unxusr
> 
> I am in the [Gitter room](https://gitter.im/commonsense/conceptnet5).

Did this ever go anywhere? I don't think the core developers are even around anymore. Everything came to a dead halt the moment Robyn Speer was no longer with Luminoso.
